vignettes/teaching1.R

```diff
@@ -101,14 +101,14 @@ autographr(callNetworkBgn, labels = FALSE, layout = "fr")
 
 # The network at half time
 callNetworkHlf <- as.matrix(callNetwork,
-                            time = calls$time[floor(nrow(calls) / 2)]) |> 
+                            time = calls$time[floor(nrow(calls) / 2)]) |>
   add_node_attribute("floor", actors$floor)
 
 autographr(callNetworkHlf, labels = FALSE, layout = "fr") +
   geom_node_point(aes(color = as.factor(floor)), size = 2, show.legend = FALSE)
 
 # The network at the end
-callNetworkEnd <- as.matrix(callNetwork, time = max(calls$time) + 1) |> 
+callNetworkEnd <- as.matrix(callNetwork, time = max(calls$time) + 1) |>
   add_node_attribute("floor", actors$floor)
 
 autographr(callNetworkEnd, labels = FALSE, layout = "fr") +
```

Comment:
Using vignettes means the plots will be changed everytime they are run, inflating the repository etc for no good reason. If they are illustrative, can they be moved to the README? If they are for learning/discovery, can they be in tutorials?